admin/base/settings/defaults.py

```diff
@@ -25,18 +25,34 @@
     '.osf.io'
 ]
 
+# Email settings. Account created for testing. Password shouldn't be hardcoded
```

Comment:
I'm sure that's fine too. I was just suggesting that maybe you don't want dev to send emails at all. That link should let you add a line to the settings file and any requested emails will just print out in the server console instead of actually sending an email. That's what OSF does when you sign up for an account. The confirmation email just prints to the server console. That line could be overwritten in the local.py for the staging and live servers.